test/sources/hatchedegg/app.mozbuild

```diff
@@ -0,0 +1,10 @@
+# vim: set filetype=python:
```

Comment:
Is it going to be a pain to keep these sources in sync with `cleanroom/source`? Do we need to keep them in sync with `cleenroom/source`?

Reply:
I hope not.  But you do have a really good point.  Motherhen's test code is strong enough to generate these files automatically as part of a test build - which means we could theoretically `git rm` these test folders and add them to `.gitignore`.  🤔 

I don't want to do this just yet, mainly because I have a few events to prepare for over the next several days.

Reply:
I'm going to defer this for now.  Two reasons:

1. Adding more complexity up front when we have a working model and this is pretty complex already
2. The test directory also acts as an example / mockup of how the user's repository should be laid out.  Removing that could be detrimental to people simply scanning the code.